.gitpod.yml

```
@@ -12,14 +12,12 @@ tasks:
     init: |
       mkdir -p .vscode
       cp tools/docker_dev/settings.json .vscode/settings.json
-      conda activate scipy-dev
+      conda activate scipy-meson
       git submodule update --init
-      python setup.py build_ext --inplace
+      python dev.py
```

```suggestion
      python dev.py --build-only
```

Also, if we have multiple CPUs on the CI, it would be better to utilize them:

```suggestion
      python dev.py --build-only -j2
```